missions/2PzD_Template.VR/customization/loadouts/German/loadoutWHR44.sqf

```diff
@@ -489,16 +465,15 @@
         params ["_unit"];
 
         [Ger_Uni_Rif_r] call Olsen_FW_FNC_AddItemRandom;
-        [Ger_Vest_HGun] call Olsen_FW_FNC_AddItem;
         [Ger_BP_Pzr] call Olsen_FW_FNC_AddItem;
         [Ger_Helmet_r] call Olsen_FW_FNC_AddItemRandom;
         [GEN_Face_r] call Olsen_FW_FNC_AddItemRandom;
 
         //Assigned Items
         GEN_Default_Equipment_Set;
 
-        //Secondary Weapon
-        WHR44_Weapon_Secondary;
+        //Primary Weapon
+        WHR44_Weapon_Rifleman;
```

Comment:
This weapon macro includes a vest, so this case doesn't need an explicit call to add a vest.